client/src/components/UserSettingField.tsx

```diff
@@ -238,7 +238,7 @@ const UserSettingFieldWithoutI18n = (props: UserSettingFieldProps) => {
             <div className="user-setting-actions">
               <Button type="submit" variant="primary" size="small" labelText={i18n._(t`Save`)} />
               <Button
-                type="submit"
+                type="button"
```

Comment:
what is the point of specifying `type="button"` on Buttons that aren't `submit` or `reset`? just wondering if i should also have been doing this